docsbox/docs/tests/test_views.py

```diff
@@ -224,32 +253,42 @@ def test_detect_convert_retrieve_file(self):
                         "fileType": "application/pdf"
                         })
                         break 
-
-                # Download file 
-                response = self.download_file(json.get("taskId"))      
-                self.assertEqual(response.status_code, 200)
-                json = ujson.loads(response.data)
-                print(json)
-                self.assertEqual(ujson.loads(response.data), {
-                    "status": "finished",
-                    "fileId": json.get("fileId"),
-                    "convertable": True,
-                    "taskId": json.get("taskId"),
-                    "fileType": "application/pdf",
-                    "fileName": file['fileName'] + ".pdf"
-                })
-                
+                # Download file nVIA
                 base_dir = os.path.abspath(os.path.dirname(__file__)+'/outputs')
-                file_dir = os.path.join(base_dir, json.get("fileName"))
-                via_response = get("{0}/{1}".format(self.app.config["VIA_URL"], json.get("fileId")), cert=self.app.config["VIA_CERT_PATH"], stream=True)     
-                self.assertEqual(via_response.status_code, 200)
+                file_dir = os.path.join(base_dir, file['fileName']+".pdf")
+                response = self.download_file(json.get("taskId"))  
+                self.assertEqual(response.status_code, 200)
                 with open(file_dir, "wb") as file:
-                    for chunk in via_response.iter_content(chunk_size=128):
-                        file.write(chunk)
+                    file.write(response.data)
                 existFile = os.path.exists(file_dir)
                 self.assertEqual(existFile, True)
                 self.assertIn(os.path.split(file_dir)[1], os.listdir(base_dir))
 
+                # Download file with VIA
```

Comment:
Can we put the VIA download behind a property? If that property is set to true, the tests are run through VIA. They will always be run without it in Travis, but it would be great if we could run it through VIA locally to help with development.

Reply:
Yes I'll check it out.